pkg/apis/auditregistration/v1alpha1/defaults.go

```diff
@@ -0,0 +1,52 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	auditregistrationv1alpha1 "k8s.io/api/auditregistration/v1alpha1"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+var (
+	// DefaultQPS is the default QPS value
+	DefaultQPS = int64(10)
+	// DefaultBurst is the default burst value
+	DefaultBurst = int64(15)
+	// DefaultThrottle is a default throttle config
+	DefaultThrottle = &auditregistrationv1alpha1.WebhookThrottleConfig{
+		QPS:   &DefaultQPS,
```

Comment:
replace with: `func(i int) *int {return &i}(DefaultQPS)`

Otherwise, someone modifying the value modifies the default for everyone. Yikes!